src/sound.c

```diff
@@ -199,17 +201,17 @@ sound_play_common(typval_T *argvars, typval_T *rettv, int playfile)
     if (context == NULL)
         return;
 
-    soundcb_T   *soundcb = get_sound_callback(&argvars[1]);
-    int         res = CA_ERROR_INVALID;
+    soundcb_T *soundcb = get_sound_callback(&argvars[1]);
+    int res = CA_ERROR_INVALID;
 
     ++sound_id;
     if (soundcb == NULL)
     {
         res = ca_context_play(context, sound_id,
-                playfile ? CA_PROP_MEDIA_FILENAME : CA_PROP_EVENT_ID,
-                tv_get_string(&argvars[0]),
-                CA_PROP_CANBERRA_CACHE_CONTROL, "volatile",
-                NULL);
+                              playfile ? CA_PROP_MEDIA_FILENAME
+                                       : CA_PROP_EVENT_ID,
```

Comment:
too much indent?

Reply:
If you notice the ? and : are aligned so you can more easily follow the ternary (at least I find that much more readable)